enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationReviewBanner/ModerationReviewBanner.unit.spec.js

```diff
@@ -24,81 +21,4 @@ describe("ModerationReviewBanner", () => {
     );
     expect(getByText("Foo verified this")).toBeTruthy();
   });
-
-  describe("when not provided an onRemove prop", () => {
```

Comment:
The ModerationReviewBanner is no longer responsible for removing verification, so the testing for the componet has reduced considerably. More tests will be added to `QuestionModerationButton`